api_tests/identifiers/views/test_identifier_list.py

```diff
@@ -480,24 +479,24 @@ def ark_payload(self):
         }
 
     @pytest.fixture()
-    def client(self):
-        return DataCiteClient(base_url='https://mds.fake.datacite.org', prefix=settings.DATACITE_PREFIX)
+    def client(self, resource):
+        return DataCiteClient(resource)
 
     @responses.activate
     def test_create_identifier(self, app, resource, client, identifier_url, identifier_payload, user,
             write_contributor, read_contributor, ark_payload):
         responses.add(
             responses.Response(
                 responses.POST,
-                client.base_url + '/metadata',
+                f'{settings.DATACITE_URL}/metadata',
```

Comment:
Why grab this from settings instead of the client? Looks like the client just isn't used at this point and can be removed from the test/class?